include/vkgcDefs.h

```diff
@@ -79,6 +79,7 @@
 //  %Version History
 //  | %Version | Change Description                                                                                    |
 //  | -------- | ----------------------------------------------------------------------------------------------------- |
+//  |     65.2 | Rename noContract in PipelineShaderOptions to noContractOpDot
```

Comment:
This is technically not a backwards-compatible change, is it? Let's bump the major version.

Reply:
Done. There should be no breakage either way as this is only used internally within the compiler.